cms/templates/js/video-thumbnail.underscore

```diff
@@ -0,0 +1,23 @@
+<div class="thumbnail-wrapper <%- action === 'upload' ? 'upload' : '' %>">
+    <img src="<%- thumbnailURL %>" alt="">
```

Comment:
We need to add an alt value here once the image is uploaded that says "Thumbnail for [file name of video]" i.e. alt="Thumbnail for fake.mp4"